src/components/Form/FormBuilder.js

```diff
@@ -19,11 +18,9 @@ function FormBuilder({
   onSubmit,
   title,
 }) {
-  const { t } = useTranslation();
-
   return (
     <Form
-      buttonLabel={buttonLabel || t('generic.form.submitLabel')}
+      buttonLabel={buttonLabel}
```

Comment:
`Form` component has a new default behavior for `Submit` button. Setting the `buttonLabel` overrides that default behavior, so taking this label out of the Formbuilder allows the buttonLabel override to tunnel down into the Form.